### PR TITLE
sp/automatic-ml-intro-w-test-drive-es-translation

### PR DESCRIPTION
**Pull request by 5675sp**
Review Spanish translation of Task 2,6, and 7.

Translated: "Next Steps"and "Appendix: Project Workspace" (The sections were still in English)

### CODE REVIEW

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 # Automatic Machine Learning Introduction with Driverless AI
 
-## Outline
+## Contorno
 
-- [Objective](#objective)
-- [Prerequisites](#prerequisites)
-- [Task 1: Product Tour](#task-1-product-tour)
-- [Task 2: Automatic Machine Learning Concepts](#task-2-automatic-machine-learning-concepts)
+- [Objetivo](#objective)
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:6]: Please update #objective to #objectivo

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+- [Requisitos Antes de Comenzar](#prerequisites)
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:7]: Please update #prerequisites to #requisitos-antes-de-comenzar

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
+- [Tarea 1: Tour del Producto](#task-1-product-tour)
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:8]: Please update #task-1-product-tour to #tarea-1-tour-del-producto

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 2 above)
Comment (ByFabrii) [reply Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:7]: Please update #Prerequisites to #Prerequisitos, a mi parecer queda más de acuerdo.

Comment (5675sp) [reply Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md]: I changed it to "prerrequisitos" es la forma de escribir _prerequisites_ en español.

**Pull request merged**